src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,308 @@
+    pal_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+
+    pal_low_frequency = 3550000000.0
+    pal_high_frequency = 3560000000.0
+
+    ppa_record_1, pal_records_1 = makePpaAndPalRecordsConsistent(ppa_record,
+                                                                 [pal_record],
+                                                                 pal_low_frequency,
+                                                                 pal_high_frequency,
+                                                                 'test_user_1')
+    # Define DPAs
+    dpa_1 = {
+       'dpaId': 'east_dpa_4',
+       'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
+    }
+    dpa_2 = {
+       'dpaId': 'east_dpa_5',
+       'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
+    }
+    dpa_3 = {
+       'dpaId': 'east_dpa_6',
+       'frequencyRange': {'lowFrequency': 3550000000, 'highFrequency': 3650000000}
+    }
+
+    # Registration and grant records for multiple iterations
+    cbsd_records_iteration_0_domain_proxy_0 = {
+        'registrationRequests': [device_1, device_2],
+        'grantRequests': [grant_request_1, grant_request_2]
+    }
+    cbsd_records_iteration_0_domain_proxy_1 = {
+        'registrationRequests': [device_3],
+        'grantRequests': [grant_request_3]
+    }
+    cbsd_records_iteration_1_domain_proxy_0 = {
+        'registrationRequests': [device_4],
+        'grantRequests': [grant_request_4]
+    }
+    cbsd_records_iteration_1_domain_proxy_1 = {
+        'registrationRequests': [device_5, device_6],
+        'grantRequests': [grant_request_5, grant_request_6]
+    }
+    
+    # Protected entities records for multiple iterations
+    protected_entities_iteration_0 = {
+        'palRecords': pal_records_1,
+        'ppaRecords': [ppa_record_1],
+        'escRecords': [esc_record_1]
+    }
+    protected_entities_iteration_1 = {
+        'gwpzRecords': [gwpz_record_1],
+        'fssRecords': [fss_record_1]
+    }
+
+    # SAS Test Harnesses configurations,
+    # Following configurations are for two SAS test harnesses for two iterations
+    sas_test_harness_device_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    sas_test_harness_device_1['fccId'] = "test_fcc_id_g"
+    sas_test_harness_device_1['userId'] = "test_user_id_g"
+
+    sas_test_harness_device_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    sas_test_harness_device_2['fccId'] = "test_fcc_id_h"
+    sas_test_harness_device_2['userId'] = "test_user_id_h"
+
+    sas_test_harness_device_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    sas_test_harness_device_3['fccId'] = "test_fcc_id_i"
+    sas_test_harness_device_3['userId'] = "test_user_id_i"
+
+    sas_test_harness_device_4 = json.load(
```

Comment:
still concerned about fcc_ids  on Sas Test Harness that SUUT does not know about, maybe we could change the cbsdSerialNumber instead and reuse known fcc_id?.  Will add an item on next week's meeting

Reply:
We can discuss this and make necessary changes if needed.